ui/src/app/enrolment/search/exam-search.component.ts

```diff
@@ -66,15 +66,12 @@ import { ExamSearchService } from './exam-search.service';
                 </div>
             }
 
-            @if (exams.length > 0) {
```

Comment:
Miksi näitä iffejä on poistettu tenttihauista?

Näyttää vähän hassulta "haku tuotti nolla osumaa" kun opiskelija ei ole edes hakenut mitään.